src/client/client.go

```diff
@@ -90,11 +93,22 @@ func collectEndpoint(req *http.Request, jsonResult interface{}) error {
 
 func ensureClient() {
 	if defaultClient == nil {
-		client, err := nrHttp.New(args2.GlobalArgs.CABundleFile, args2.GlobalArgs.CABundleDir, time.Second*30)
+		clientOptions := []nrHttp.ClientOption{
+			nrHttp.WithTimeout(DefaultTimout),
+		}
+		if args2.GlobalArgs.CABundleDir != "" {
```

Comment:
is this due to infra dependency bump? `nrHttp.New` was no longer available?

Reply:
yes exactly, it is part of the sdk dependency bumping